app/views/locations/deactivate_confirm.html.erb

```diff
@@ -4,7 +4,7 @@
      <% end %>
    <h1 class="govuk-heading-l">
      <span class="govuk-caption-l"><%= @location.postcode %></span>
-     This change will affect <%= @location.lettings_logs.count %> logs
+     This change will affect <%= @location.lettings_logs.filter_by_before_startdate(@deactivation_date).count %> logs
```

Comment:
Why don't we just hide this line if there is no impact? I think we should still give people an opportunity to back out.

Reply:
There was a discussion in the ticket and this was the decision

Reply:
Okay! Can we memoize this query? We're executing it twice.